packages/scm/src/browser/scm-preferences.ts

```diff
@@ -23,6 +23,8 @@ import {
     PreferenceContribution
 } from '@theia/core/lib/browser/preferences';
 
+export type VIEW_MODE = 'tree' | 'list';
```

Comment:
I believe wrapping it in a namespace would might make usage of it clearer:

```ts
export namespace ScmPreferences {
  export type ViewMode = 'tree' | 'list';
}
```

It can then be used like so:

```ts
const registerToggleViewItem = (command: Command, mode: ScmPreferences.ViewMode) => { ...
```